src/main/java/com/google/sps/data/TimeZoneInfo.java

```diff
@@ -0,0 +1,34 @@
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.TimeZone;
+
+public class TimeZoneInfo {
+
+  private static final double MILLISECONDS_TO_HOURS = 3600000.0;
```

Comment:
nit but better to name it something that makes it clear what direction the conversion is in.

MILLISECONDS_PER_HOUR for eg. makes it clear.